media/js/firefox/whatsnew/whatsnew-77-zhCN.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    function trackVideoInteraction(title, state) {
+        window.dataLayer.push({
+            'event': 'video-interaction',
+            'videoTitle': title,
+            'interaction': state
+        });
+    }
+
+    function initVideoInteractionTracking() {
+        var video = document.getElementById('pip-video');
+
+        video.addEventListener('play', function() {
+            trackVideoInteraction(this.getAttribute('data-video-title'), 'video play');
```

Comment:
The title is coming through as `undefined` in the dataLayer event. Should this be `data-ga-label`?
```suggestion
            trackVideoInteraction(this.getAttribute('data-ga-label'), 'video play');
```

Reply:
Oops, actually the attribute should be changed in the HTML. Copypasta error 🍝